bin/sl-meshctl.js

```diff
@@ -144,6 +144,12 @@ function cmdStatus(instance) {
     fmt(1, 'port', '%s', rsp.port);
     fmt(1, 'base', '%s', rsp.base);
 
+    if (rsp.version) {
+      fmt(1, 'version');
+      fmt(2, rsp.version.name, 'v%s', rsp.version.version);
```

Comment:
Now these object paths just look weird.. maybe version should just be a flat object and the UI can just dump it as key: value...

```js
rsp.version = {
  api: require('strong-mesh-models/package.json').version,
  'strong-pm': require('../package.json').version,
};
```

Leave the `v` prefix out of the printing at let the server put it in, that way it is free to put free-form strings in place of these values if desired.
```js
fmt(1, 'version');
Object.keys(rsp.version).forEach(function(k) {
  fmt(2, '%s: %s', k, rsp.version[k]);
});
```